client/app/domain/domain-tabs.controller.js

```diff
@@ -131,10 +131,7 @@ angular.module('App').controller(
     }
 
     static getUpdateOwnerUrl(domain) {
-      if (
-        _.get(domain, 'whoisOwner', false)
-        && domain.whoisOwner !== 'pending'
-      ) {
+      if (_.isObject(domain.whoisOwner) && _.has(domain, 'whoisOwner.id')) {
         return `#/useraccount/contact/${domain.name}/${domain.whoisOwner.id}`;
```

Comment:
You should also check if `domain` has a `name`

Reply:
This check has been added